app/views/audits/index.html.erb

```diff
@@ -26,7 +26,7 @@
           <td class="govuk-table__cell"> User deleted </td>
         <% end %>
         <td colspan="2" class="govuk-table__cell">
-          <% if audit_item.activity_type.include?("request") %>
+          <% if audit_item.activity_type.include?("request") || audit_item.activity_type.include?("document_received_at_changed") %>
```

Comment:
@freesteph I've changed it back to the previous version that doesnt use regex and works fine. Your suggestion doesn't work as in? cannot account for partial string matches so it'd always need an extra iteration - with any? for ex to match the two strings, which seems silly to check just two conditions.

Reply:
in does account partial matches

```ruby
# in a rails console
"request".in? "foobar_request"
=> true
```

Reply:
steph, again, your example doesn't work - in? is unable to check against two values, because it only takes a single string for a partial match, it can't do a partial match on two strings - it looks for the exact string. 

You don't need to illustrate something I already spent a long time trying with different variations because I really wanted to make you happy and put in that example. But it doesn't work. I know that a single string can be checked - I have tried your example, I have tried it with every possible iteration and I wish that you would trust me and try it with the example we need - which is checking against two strings - and then show me that it works:

![image](https://user-images.githubusercontent.com/9452321/143566050-8443cf0f-e4d6-4dee-ad2c-c4782c16d7d0.png)
![image](https://user-images.githubusercontent.com/9452321/143566172-e71b42e0-b333-4c24-a3c2-94727154080b.png)

and if you swap them around, and have it check against the description_change_value, it can't check the partial match again:
![image](https://user-images.githubusercontent.com/9452321/143566511-655cfa3e-8355-4fbe-b1fb-00615c15357b.png)
![image](https://user-images.githubusercontent.com/9452321/143566571-14370e09-e931-4e9b-b89c-f35795cfc45e.png)

Reply:
That's my bad, I'm really sorry. I got confused about what matches on which side of `in?` and `include?`. Very, very sorry.